server/container_stop.go

```diff
@@ -81,7 +81,8 @@ func (s *Server) stopContainer(ctx context.Context, ctr *oci.Container, timeout
 
 	if hooks != nil {
 		if err := hooks.PostStop(ctx, ctr, sb); err != nil {
```

Comment:
@haircommander Hey, the error fix is needed for sure, but I was actually wondering if the hook should move below the stopContainer call.

Reply:
that's nri.stopContainer, the actual container stopping has been done already

Reply:
Ack.